SwiftLeeds/Views/Local/LocalViewModel.swift

```diff
@@ -8,23 +8,35 @@
 import Foundation
 import NetworkKit
 import SwiftUI
+import MapKit
 
 class LocalViewModel: ObservableObject {
+    @Published var selectedCategory: Local.LocationCategory?
+    @Published var categories: [Local.LocationCategory] = []
+
     @Environment(\.network) var network: Networking
 
-    var categories: [Local.LocationCategory] = []
     var error: Error?
     
     func loadData() async {
         do {
             let localResults = try await network.performRequest(endpoint: LocalEndpoint())
-            self.categories = localResults.data
+
+            DispatchQueue.main.async {
+                self.categories = localResults.data
+                self.selectedCategory = self.categories.first
+            }
         } catch {
             self.error = error
         }
+
         await publishResults()
     }
 
+    var annotationItem: Local.Location {
```

Comment:
Is that a....force unwrap 👀

Reply:
That was used for initial testing of the Map, taken out